### DISCUSSION TITLE
Home Assistant 2023.8.0 - MQTT entities name changes

### DISCUSSION THREAD

**Discussion by N1co93** [tsightler/ring-mqtt#698] [Support Q&A]
Starting with Home Assistant 2023.8.0 the MQTT entities name changes. Therefore I get the following warning, after starting my Home Assistant:

``` 
Some MQTT entities have an entity name that starts with the device name. This is not expected. To avoid a duplicate name the device name prefix is stripped of the entity name as a work-a-round. Please inform the maintainer of the software application that supplies the affected entities to fix this issue.

List of affected entities:

- lock.haustur_lock
- sensor.haustur_info
- camera.staubi_rendered_map
- binary_sensor.haustur_ding
- sensor.haustur_battery
- sensor.haustur_wireless
```
![image](https://github.com/tsightler/ring-mqtt/assets/59510296/a12bc49d-b27b-486e-826f-47f5b0759379)

You can find more information in this Home Assistant webpages:
https://developers.home-assistant.io/blog/2023-057-21-change-naming-mqtt-entities/
https://community.home-assistant.io/t/psa-mqtt-name-changes-in-2023-8/598099

**Comment by tsightler** [comment]
Hi all.  I'm not usually one to moderate too much, but I've deleted the comments in this thread since mostly is was me complaining, so consider this an attempt to self-moderate, some others might be caught in the wash though.  I appreciate those that commented with messages of support or with technical notes to help, they are much appreciated.

I'm going to close and lock this thread for now, but the official issue to track this is at https://github.com/tsightler/ring-mqtt/issues/700.  I'm sure I'll have a fix for it by the time 2024.2 rolls around, probably much sooner.
